### PR TITLE
add jupyter-(docprovider|collaboration-ui) 1.0.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

References:
- unblocks https://github.com/conda-forge/jupyter-collaboration-feedstock/pull/18

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/help-python @conda-forge/jupyter-collaboration ready for review

**Review by davidbrochart — APPROVED** [review]
Thanks.

File: recipes/jupyter-collaboration-ui/meta.yaml
Comment (ocefpaf) [inline recipes/jupyter-collaboration-ui/meta.yaml:2]: Do we really need this? It will became a complication as we move to the new recipe format.

Comment (bollwyvl) [reply recipes/jupyter-collaboration-ui/meta.yaml:2]: Do I need the version so people don't get broken stuff? Yes. Does it need the `if`? No, but I'm tired of fixing bot dupes, and this seems to confuse it enough that it doesn't touch things it shouldn't.

If the new format is preferred, I can just use that instead... last time I excitedly tried, i was told it wasn't ready to be used (even though the example was in here)....

Comment (ocefpaf) [reply recipes/jupyter-collaboration-ui/meta.yaml:2]: We are in slowly transition. I would not merge this if it was another maintainer but I'm sure you will be active and update this when the new format rolls out... Still I recommend removing as much jinja as possible from your existing and future recipes.

Comment (bollwyvl) [reply recipes/jupyter-collaboration-ui/meta.yaml:2]: > removing as much jinja as possible

Oh, that's my goal when living that conda-forge life. I'd use native YAML anchors, but multi-outputs _hate_ that. But having made enough "forgot to update a python version" errors, DRY wins. I think the new format will improve the situation (even though I think it could have used more existing techniques like JSON-e, patch, etc. and don't like that it doesn't declare a `$schema` for completion) but it's a big net step forward, and the significant speed improvements are certainly welcome!

**Pull request merged**
